hack/dockerfile/install/golangci_lint.installer

```diff
@@ -1,11 +1,19 @@
 #!/bin/sh
 
-: "${GOLANGCI_LINT_COMMIT=v1.17.1}"
+: "${GOLANGCI_LINT_COMMIT=v1.20.0}"
 
 install_golangci_lint() {
 	echo "Installing golangci-lint version ${GOLANGCI_LINT_COMMIT}"
 	go get -d github.com/golangci/golangci-lint/cmd/golangci-lint
 	cd "$GOPATH/src/github.com/golangci/golangci-lint/" || exit 1
```

Comment:
side note; maybe we should do `set -u -e` at the beginning of this script, instead of using `|| exit 1` here.

Reply:
I think we can in-line most of these scripts in the Dockerfile eventually